tests/static_lib_arch_flags_configs/lib_static0/lib_static0/lib_build_info.cmake

```diff
@@ -21,12 +21,12 @@ set(LIB_ARCHIVE_FLAGS_static0_archive1 -DLIB_STATIC0_OPTION_archive1=1)
 set(LIB_DEPENDENT_MODULES "")
 
 # Specify includes for building archives */
-set(LIB_ARCHIVE_INCLUDES api)
+set(LIB_ARCHIVE_INCLUDES inc)
 
 ### Items to allow use of this lib (contained archives and additonal source files)
 
 # Include directory to expose to the dependent libs/apps
-set(LIB_INCLUDES api)
+set(LIB_INCLUDES api inc)
```

Comment:
Okay, this is why the include of static0_archive.h etc works above.

This change should be undone.

Reply:
Reverted.